src/common/components/header.tsx

```diff
@@ -4,10 +4,9 @@ import * as React from 'react';
 
 import { HeaderIcon, HeaderIconDeps } from 'common/components/header-icon';
 import { NamedFC } from 'common/react/named-fc';
-import { title } from 'content/strings/application';
 import * as styles from './header.scss';
 
-export type HeaderDeps = HeaderIconDeps;
+export type HeaderDeps = { applicationTitle: string } & HeaderIconDeps;
```

Comment:
This seems to have your changes from the other PR?

Reply:
Yes. It's likely they are combined. I will remerge with master to clean that up.

Reply:
Addressed.